go/test/endtoend/onlineddl/vtgate_util.go

```diff
@@ -106,6 +106,12 @@ func CheckCancelAllMigrations(t *testing.T, vtParams *mysql.ConnParams, expectCo
 	assert.Equal(t, expectCount, int(r.RowsAffected))
 }
 
+// ReadMigrations reads migration entries
+func ReadMigrations(t *testing.T, vtParams *mysql.ConnParams, like string) *sqltypes.Result {
+	showQuery := fmt.Sprintf("show vitess_migrations like '%s'", like)
```

Comment:
Can I talk you into using `sqlparser.TrackedBuffer` for this? I don't think anybody is gonna do SQL injection in our integration test suite buuuut it's a good consistency habit to pick up :p

Reply:
Sure thing

Reply:
I'm looking at Vitess code and I don't see `sqlparser.TrackedBuffer` widely used for encapsulating values; I see construction using `sqlparser.NewStrLiteral`, like in:

https://github.com/vitessio/vitess/blob/bb15f60111dd984fcee83a392ac5b97d260ba125/go/vt/vtgate/planbuilder/show.go#L217-L226

Any thoughts on how to use `TrackedBuffer` to avoid SQL injection?

Reply:
Rewritten to use `sqlparser.ParseAndBind()`

Reply:
That's the one! Sorry I didn't make it explicit.

Reply:
Good good, thank you!